### PR TITLE
update README to include seeing it action

### PR DESCRIPTION
**Pull request by brucearctor**

### CODE REVIEW

**Comment by brucearctor** [comment]
@matthewrobertson -- I don't have enough visibility to fix  the failure --> 
`import/copybara — An error happened while migrating the change` 
... ?

File: README.md [hunk 1]
```diff
@@ -12,26 +12,33 @@ Google Cloud's container platforms:
 ## Quickstart
 
 1. [Install Docker](https://store.docker.com/search?type=edition&offering=community)
-1. [Install the pack tool (a CLI for running Buildpacks)](https://buildpacks.io/docs/install-pack/)
-1. Clone the [sample apps](https://github.com/GoogleCloudPlatform/buildpack-samples):
+2. [Install the pack tool (a CLI for running Buildpacks)](https://buildpacks.io/docs/install-pack/)
```
Comment (grant) [inline README.md:15]: Can you keep using `1.` throughout? No need to manually increment.

**Review by grant — CHANGES_REQUESTED** [review]

**Comment by grant** [comment]
Merged with https://github.com/GoogleCloudPlatform/buildpacks/commit/4f5fb95a2af6887fbe4302be81c401d18902f260.

File: README.md (hunk 1 above)
Comment (brucearctor) [reply README.md:15]: eI can submit a new PR to make everything 1.

**Pull request closed without merging**